nautobot/virtualization/models.py

```diff
@@ -345,11 +345,11 @@ class VMInterface(PrimaryModel, BaseInterface):
         on_delete=models.CASCADE,
         related_name="interfaces",
     )
-    name = models.CharField(max_length=64, db_index=True)
+    name = models.CharField(max_length=255, db_index=True)
     _name = NaturalOrderingField(
         target_field="name", naturalize_function=naturalize_interface, max_length=100, blank=True, db_index=True
```

Comment:
Missed `max_length` here, but hm, this may be tricky. A `NaturalOrderingField` ideally should be able to store more characters than its `target_field` since it will expand numeric strings with additional digits (e.g. `"Gi1/2/3/4/5:6.7"` becomes `"0001000200030004Gi000005000006000007"` per `naturalize_interface()`), but MySQL doesn't like `db_index=True` with `max_length` greater than 255. I guess we just set this one to 255 as well, and rely on the fact that `NaturalOrderingField` will truncate naturalized strings if they exceed its max_length?